wrappers/obj-c/ODWLogManager.mm

```diff
@@ -52,6 +55,47 @@ +(nullable ODWLogger *)loggerWithTenant:(nonnull NSString *)tenantToken
     return [[ODWLogger alloc] initWithILogger: logger];
 }
 
++(nullable ODWLogger *)loggerWithTenant:(nonnull NSString *)tenantToken
```

Comment:
This method  is short-circuiting LogManager initialization logic (look at initializeLogManager method), resulting not only on code duplication but also skiping Log::ManagerInitialize, which is required. Try to create an implementation that reuses existing initialization logic as much as possible

Reply:
The issue is we do not want to update the static functions here, and it looks like most/all of the code in the `initializeLogManager` is using the static `LogManager` functions

Reply:
@theta3 I noticed there is an existing `initForTenant:withConfig:` [function](https://github.com/microsoft/cpp_client_telemetry/blob/master/wrappers/obj-c/ODWLogManager.mm#L55) here. Would we be able to use that instead of this new function?

Reply:
According to your scenario you are getting a second logger, so I would assume consumer called an existing loggerWithTenant method already, and you should at minimum for _initialized and fail otherwise. If my assumption is incorrect then initialization should not be skipped, because also things like turning off statistics, and setting semantics for AppVersion, AppLanguage and UserLanguage happens here

Reply:
Answering your question on "I noticed there is an existing... ", I think calling this it would override existing default configuration, which it's not what you are looking for

Reply:
That's a good point @eduardo-camacho.  That said, I'm not sure I see the reason why there needs to be a static instance of the configuration at all.  It seems to make sense to pass the configuration into the LogManager, and have the LogManager instance be associated 1-to-1 with the specified configuration.   Would you mind clarifying the intent there?